psi-sample/host/build.gradle

```diff
@@ -1,18 +1,26 @@
 plugins {
     id 'application'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
 }
 
-mainClassName = "com.r3.conclave.sample.host.Host"
+mainClassName = "com.r3.conclave.host.web.EnclaveWebHost"
 
 // Override the default (simulation) with -PenclaveMode=
-def mode = findProperty("enclaveMode")?.toString()?.toLowerCase() ?: "simulation"
+def mode = findProperty("enclaveMode")?.toString()?.toLowerCase() ?: "mock"
 
 dependencies {
     implementation "com.r3.conclave:conclave-host:$conclaveVersion"
 
     // We need the enclave at runtime but not when compiling the host, because they don't share types.
     runtimeOnly project(path: ":enclave", configuration: mode)
     runtimeOnly "org.slf4j:slf4j-simple:1.7.30"
```

Comment:
This isn't needed anymore as `conclave-web-host` already adds it

Reply:
you mean runtimeOnly "org.slf4j:slf4j-simple:1.7.30" right?